server/src/main/scala/za/co/absa/atum/server/api/http/Endpoints.scala

```diff
@@ -47,4 +47,14 @@ trait Endpoints extends BaseEndpoints {
       .out(jsonBody[AtumContextDTO])
   }
 
+  protected val createOrUpdateAdditionalDataEndpoint
+    : PublicEndpoint[AdditionalDataSubmitDTO, ErrorResponse, AdditionalDataSubmitDTO, Any] = {
+    apiV1.post
+      .name(CreateOrUpdateAdditionalData)
+      .in(CreateOrUpdateAdditionalData)
```

Comment:
Here the `.name` is used to describe the name of the endpoint which is used in the url of its swagger documentation. It can be omitted but the url would be different. Compare the two cases below. 
```
http://localhost:8080/docs/#/default/createOrUpdateAdditionalData
http://localhost:8080/docs/#/default/postApiV1Createorupdateadditionaldata
```
So basically by doing the `.name` we make the url same as the actual endpoint name. 

The `.in` describes the api path. 

Notice our endpoints are based off of this `apiV1` base endpoint. 
```scala
protected val apiV1: PublicEndpoint[Unit, ErrorResponse, Unit, Any] = {
    baseEndpoint.in(Api / V1)
}
```

To that we add the next part which in this case is the content of the `CreateOrUpdateAdditionalData` constant, the "createOrUpdateAdditionalData" respectively; and we end up with `api/v1/createOrUpdateAdditionalData`.

Reply:
Maybe we could actually omit the name. What do you think? Do we care about the link of the docs page? @lsulak @benedeki

Reply:
I am sorry for being slow, but I am not getting it. I would ask for a call explaining it.
(Of course, I don't find this comment a problem. I just want to learn from it)

Reply:
@lsulak 
We agreed on call with David to remove the `.name` from all endpoints.

Reply:
okay then, I'll drop it